webapp/app/_components/footer/footer.tsx

```diff
@@ -0,0 +1,109 @@
+import Image from "next/image";
+import data from "./footerLinks";
+
+const Footer = () => {
+  return (
+    <>
+      <section className="text-[#CFCDD2] bg-[#221932] px-24 py-16">
+        <div className="flex justify-between ">
+          <div className="text-3xl w-2/5 ">
+            <p className="pb-4 font-semibold">Resume Matcher</p>
+            <ul className="flex ">
+              <a
+                className="text-base pr-4 hover:text-[#A49DA2]"
+                href="https://github.com/srbhr/Resume-Matcher"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Github
+              </a>
+              <a
+                className="text-base hover:text-[#A49DA2]"
+                href="https://github.com/srbhr/Resume-Matcher/blob/master/LICENSE"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Apache 2.0
+              </a>
+            </ul>
+          </div>
+          <div className="flex justify-around text-xl w-3/5">
+            {data.map((section, index) => (
+              <div key={index}>
+                <p className="pb-4 font-semibold">{section.label}</p>
+                <ul>
+                  {section.items.map((item, itemIndex) => (
+                    <li
+                      className="py-1 text-base hover:text-[#A49DA2]"
+                      key={itemIndex}
+                    >
+                      <a
+                        {...(item.shouldOpenInNewPage
+                          ? { target: "_blank" }
+                          : {})}
+                        rel="noopener noreferrer"
+                        href={item.link}
+                      >
```

Comment:
👀 We could further refactor the dynamic conditional logic to be simpler to read and more maintainable with the following suggestion below, which applied to both the `target` and `rel` attributes.

```suggestion
                      <a
                        target={item.shouldOpenInNewPage ? "_blank" : undefined}
                        rel={
                          item.shouldOpenInNewPage
                            ? "noopener noreferrer"
                            : undefined
                        }
                        href={item.link}
                      >
```

Reply:
Thanks for your valuable suggestions @Sayvai 🙏

Reply:
Fixed all the changes suggested by @Sayvai